common/src/test/java/io/seata/common/thread/NamedThreadFactoryTest.java

```diff
@@ -65,4 +65,11 @@ public void testThreadNameHasCounterWithPrefixCounter() {
                 .isEqualTo(thread.getName());
         }
     }
+    @Test
```

Comment:
Add an empty line before @Test would be better.

Reply:
Thank you for your advice，I've dealt with it

Reply:
> Add an empty line before @test would be better.
Thank you for your advice，I've dealt with it

Reply:
> Add an empty line before @test would be better.
Thank you for your suggestion. I have dealt with it

Reply:
Thank you for your suggestion. I have dealt with it

Reply:
> Add an empty line before @test would be better.

Thank you for your suggestion. I have dealt with it